assets/app/scripts/controllers/edit/buildConfig.js

```diff
@@ -0,0 +1,307 @@
+      },
+      {
+        title: $routeParams.buildconfig,
+        link: "project/" + $routeParams.project + "/browse/builds/" + $routeParams.buildconfig
+      },
+      {
+        title: "Edit"
+      }
+    ];
+
+    $scope.buildFrom = {
+      projects: [],
+      imageStreams: [],
+      tags: {},
+    }
+    $scope.pushTo = {
+      projects: [],
+      imageStreams: [],
+      tags: {},
+    }
+
+    $scope.triggers = {
+      "webhook": false,
+      "imageChange": false,
+      "configChange": false
+    };
+
+    var watches = [];
+
+    ProjectsService
+      .get($routeParams.project)
+      .then(_.spread(function(project, context) {
+        $scope.project = project;
+        DataService.get("buildconfigs", $routeParams.buildconfig, context).then(
+          // success
+          function(buildConfig) {
+            $scope.buildConfig = buildConfig;
+            $scope.updatedBuildConfig = angular.copy($scope.buildConfig);
+            $scope.buildStrategy = $filter('buildStrategy')($scope.updatedBuildConfig);
+            $scope.envVars = $filter('envVarsPair')($scope.buildStrategy.env);
+
+            angular.forEach($scope.buildConfig.spec.triggers, function(value, key) {
+              switch (value.type) {
+                case "Generic":
+                  break;
+                case "GitHub":
+                  $scope.triggers["webhook"] = true;
+                  break;
+                case "ImageChange":
+                  $scope.triggers["imageChange"] = true;
+                  break;
+                case "ConfigChange":
+                  $scope.triggers["configChange"] = true;
+                  break;
+              }
+            });
+            $scope.pickedTriggers = $scope.triggers;
+            
+
+            var buildFrom = $scope.buildStrategy.from;
+            var testFrom = $filter('imageObjectRef')($filter('buildStrategy')($scope.buildConfig).from, buildConfig.metadata.namespace)
+            var pushTo = $filter('imageObjectRef')(buildConfig.spec.output.to, buildConfig.metadata.namespace)
+
+            $scope.options = {
+              pickedBuildFromType: buildFrom.kind,
+              pickedBuildFromNamespace: buildFrom.namespace || $scope.projectName,
+              pickedBuildFromImageStream: buildFrom.name.split(":")[0],
+              pickedBuildFromImageStreamTag: buildFrom.name.split(":")[1],
+
+              pickedPushToNamespace: buildConfig.spec.output.to.namespace || buildConfig.metadata.namespace,              
+              pickedPushToImageStream: buildConfig.spec.output.to.name.split(":")[0],
+              pickedPushToImageStreamTag: buildConfig.spec.output.to.name.split(":")[1],
+
+              pickedBuildFromDockerImage: $filter('buildStrategy')($scope.updatedBuildConfig).from.name || "",
+            };
+
+            $scope.options.forcePull = false;
+            if ($scope.buildStrategy.forcePull == true) {
+              $scope.options.forcePull = true;
```

Comment:
```
$scope.options.forcePull = $scope.buildStrategy.forcePull;
```

or if you need to make sure it's a boolean

```
$scope.options.forcePull = !!$scope.buildStrategy.forcePull;
```

This could be set in the `$scope.options` declaration above.